priv/templates/nimble_template/test/otp_app_web/plugs/health_plug_test.exs.eex

```diff
@@ -0,0 +1,64 @@
+defmodule <%= web_module %>.HealthPlugTest do
+  use <%= web_module %>.ConnCase, async: true
+
+  import ExUnit.CaptureLog
+
+  alias <%= web_module %>.HealthPlug
+
+  describe "init/1" do
+    test "returns given options" do
+      assert HealthPlug.init([]) == []
+    end
+  end
+
+  describe "call/2" do
+    test "returns 200 status given the liveness path" do
```

Comment:
Should we update the test description to align with the new convention 💡 
```suggestion
    test "given the liveness path, returns 200 status" do
```

Reply:
adjusted it in https://github.com/nimblehq/elixir-templates/pull/159/commits/06a24ca05d89961279d29d39e0542e117dafe7c4 @rosle